libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -294,6 +294,7 @@
         "createHeader": "Create actor",
         "errorToastLabel": "Error",
         "masterDataTabHeader": "Master data",
+        "meteringpointTypesHeader": "Meteringpoint types",
```

Comment:
nitpick:suggestion: in other places in this file "Meteringpoint" is written in two words. Let's keep it consistent. Similarly, let's change the key to `meteringPointTypesHeader` (also in `da.json` file).